src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  const adder = (numberToAdd) => {
+    if (numberToAdd != null) {
```

Comment:
why comparing with null? and not strong comparing.
```suggestion
    if (numberToAdd !== undefined) {
```